### DISCUSSION TITLE
Infinite Dependencies install

### DISCUSSION THREAD

**Discussion by gamingoffice** [hlky/stable-diffusion-webui#793] [Q&A]
![image](https://user-images.githubusercontent.com/109173604/188978898-ea8ace43-2bef-4efc-80a4-39f3f661c4e1.png)

Its installing pip dependencies for like ages its like infinitly is this a bug or anyone experienced the same?

**Comment by Joeyd348039** [comment]
Same. Looking for a solution but no luck

**Reply by raga-create** [reply]
I apologize in advance if this is confusing but I know almost nothing about the things I did. However, I want to share the steps I used to get past this issue since I wont be the only user with limited expertise who runs into this. 

I followed the installation guide in the wiki, in hindsight it contains some information that is probably related to this. 
My first mistake was to create the "src" folder and subfolders for the model before running the webui.cmd. I assume this stops the installation to create subfolders or something similar. So I renamed the folders I created in src by adding a 2 to their name. 

It still didnt work at this point, so I decided to tinker even more: 

If you open the anaconda prompt, you can switch to the created environment by entering "conda activate ldm". Navigate to the folder where the webui.cmd is located and type "python "%CD%"\scripts\relauncher.py" (taken from the webui.cmd). This bypasses the pip dependency installation (i think). It is something I got from [this post](https://github.com/sd-webui/stable-diffusion-webui/discussions/848https://github.com/sd-webui/stable-diffusion-webui/discussions/848).
I also entered set PIP_EXISTS_ACTION=w into my command prompt at some point, following the [suggestion on reddit](https://www.reddit.com/r/deepdream/comments/ww03vn/comment/io2pbdc/?utm_source=share&utm_medium=web2x&context=3).  This might have helped or not, I really cant tell. 

Now it works without issues, even with the webui.cmd. If you have any questions, I will try to help.
